extensions/amp-carousel/0.1/slidescroll.js

```diff
@@ -479,6 +499,11 @@ export class AmpSlideScroll extends BaseSlides {
     this.slideIndex_ = newIndex;
     this.hideRestOfTheSlides_(showIndexArr);
     this.setControlsState();
+
+    if (opt_isUserAction) {
```

Comment:
i think we should make this var something like , `!isBindAction` it would be hard to quantify what is userAction right?